tests/cli/lib/lib.sh

```diff
@@ -16,6 +16,27 @@ if [ "$COMPOSER_TEST_FAIL_FAST" == "1" ]; then
   }
 fi
 
+export QEMU="qemu-system-$(uname -m)"
```

Comment:
This is missing the additional qemu options from the test scripts before the cherry-pick. Just copy the deleted lines and paste them here.

Reply:
OK, I changed the QEMU* variables to conform to the previous state (before Brian's commit), hope it's fine this way.